ci_framework/roles/cifmw_cephadm/tasks/rgw.yml

```diff
@@ -0,0 +1,44 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+- name: Collect the host and build the resulting host list
+  set_fact:
+    _hosts: "{{ _hosts|default([]) + [ hostvars[item]['ansible_hostname'] ] }}"
+  with_items: "{{ groups['edpm'] }}"
```

Comment:
using `loop: "{{ groups['edpm'] }"` is "the modern way" of looping

Reply:
ack changing that accordingly

Reply:
Done